### PR TITLE
Add Docker & CircleCI & Godeps config and dependencies

### PR DESCRIPTION
**Pull request by macat**
- Adding docker-compose & dockerfiles
- Godeps
- CircleCI config

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package main
+
+import "testing"
+
+func TestMain(t *testing.T) {
+	if true != true {
+		t.Error("test failed")
+	}
+}
```
Comment (bobbytables) [inline main_test.go]: What is this test file for?

Comment (macat) [inline main_test.go]: Whitenoise. I'll remove it.

**Comment by poopoothegorilla** [comment]
+1

**Comment by bobbytables** [comment]
:+1:

**Pull request merged**